src/layouts/section/cp-k8s-api-reference.html

```diff
@@ -0,0 +1,35 @@
+{{ partial "header.html" . }}
+
+<div class="row">
+  <div class="col-md-10 col-md-offset-1">
+    <h1 id="title">Control Plane Kubernetes API Reference</h1>
+
+    <p>The Control Plane Kubernetes API gives you access to the custom resources (CR) defining your tenant clusters in your Giant Swarm installation’s control plane. Here you find schema descriptions for the custom resources in use.</p>
+
+    <p>This is an early preview of our Control Plane Kubernetes API documentation, focusing on Custom Resource (CR)/Custom Resource Definition (CRD) schema.</p>
+
+    <p>Watch this space in the upcoming weeks to find improvements and more documented details. And feel free to give us any <strong>feedback</strong> and let us know how we could do better!</p>
+
+    <hr />
+
+    <h2>Custom Resource Definitions</h2>
+
+    <p>Select the CR/CRD name to get information on the custom resource schema.</p>
+
+    <ul class="linklist" style="margin-top: 2em; margin-bottom: 2em;">
```

Comment:
Can we please extract these inline styles to `crd.css`?

Reply:
Yes, will change